uui/components/widgets/Badge.module.scss

```diff
@@ -255,6 +279,10 @@
         }
     }
 
+    :global(.uui-count_indicator):last-child {
+        margin-right: 0;
```

Comment:
what it the case for this rule?

Reply:
when CountIndicator last and no Icon after it.

Reply:
@siarhei-epam do we need this rule? i guess that count indicator can't be the last child in badge, because we always have a caption after it